dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/AssemblyCheckFilterProcessorTest.java

```diff
@@ -0,0 +1,191 @@
+                                                                            1325045L, 1325046L, Orientation.REVERSE,
+                                                                            null, "11828");
+
+        assertNotNull(assemblyChecker.process(insertionWithValidReference));
+    }
+
+    @Test
+    public void validDeletion() throws Exception {
+        SubSnpCoreFields deletionWithValidReference = new SubSnpCoreFields(1545663038L, Orientation.FORWARD, 738583051L,
+                                                                           Orientation.REVERSE, "NT_455997.1", 1330026L,
+                                                                           1330028L, Orientation.REVERSE,
+                                                                           LocusType.DELETION, "22", 12765L, 12767L,
+                                                                           "ACA", "ACA", null, "ACA/-",
+                                                                           "NC_006109.4:g.12765_12767delACA", 12765L,
+                                                                           12767L, Orientation.FORWARD,
+                                                                           "NT_455997.1:g.1330026_1330028delTGT",
+                                                                           1330026L, 1330028L, Orientation.REVERSE,
+                                                                           null, "1062064");
+
+        assertNotNull(assemblyChecker.process(deletionWithValidReference));
+    }
+
+    @Test
+    public void invalidDeletion() throws Exception {
+        SubSnpCoreFields deletionWithInvalidReference = new SubSnpCoreFields(1545663038L, Orientation.FORWARD,
+                                                                             738583051L, Orientation.REVERSE,
+                                                                             "NT_455997.1", 1330026L, 1330028L,
+                                                                             Orientation.REVERSE, LocusType.DELETION,
+                                                                             "22", 12765L, 12767L, "ATA", "ATA", null,
+                                                                             "ATA/-", "NC_006109.4:g.12765_12767delATA",
+                                                                             12765L, 12767L, Orientation.FORWARD,
+                                                                             "NT_455997.1:g.1330026_1330028delTAT",
+                                                                             1330026L, 1330028L, Orientation.REVERSE,
+                                                                             null, "1062064");
+
+        assertNull(assemblyChecker.process(deletionWithInvalidReference));
+    }
+
+    
+    @Test
+    public void validMnv() throws Exception {
```

Comment:
I think this is considered an INDEL, an MNV would be something like TC -> AG (same length).
Also applies to method `invalidMnv`.